### DISCUSSION TITLE
New JLPCB order - who wants to make a pool?

### DISCUSSION THREAD

**Discussion by mikenturner** [cardonabits/haxo-hw#16] [General]
Hi
I've just come across this project and I'm very interested to make a haxohone, but as has been stated in a number of places the minium irder is 5, so does anybody else want to order any of the other 4?  
Thanks
Mike
PS: I'm based in the UK

**Comment by jcard0na** [comment]
Hi Mike,

You might check with @raychmond  He [built his own haxophone](https://github.com/jcard0na/haxo-hw/discussions/15) recently and might have an extra PCB left over he might want to sell you.

Cheers,
Javier

**Comment by bonjourrichard** [comment]
Hi Mike,
I'd be interested if we can get a few other people.   I am located in the USA.

Best, Richard

**Reply by mikenturner** [reply]
Hi Richard 
great to hear from you.  Currently I only have one other person interested (also UK based like me), so I would plan to get the PCBs delivered to the UK.  But if you are willing to pay for me to post one to you in the USA, I'll add you to the pool.
Mike

**Reply by bonjourrichard** [reply]
Hi Mike,
Thanks for creating this opportunity and offering to send the PCBs to me in the USA.  However, I am going to delay my plan to build the wonderful haxophone.  I want to build one with my sons, and they aren't yet ready for the challenge.

Please post photos of your completed haxophone.  I will be following this project until my sons are ready to help me.

Best, Richard

**Comment by mikenturner** [comment]
For your info, I found someone else who was willing to pay for 2 boards, so I went ahead and placed an order, and the PCBs arrived today.  So if anyone is interested I now have some boards in the UK

Mike
![PCBs](https://github.com/jcard0na/haxo-hw/assets/60625283/baba9698-ff50-4553-b679-089303b5dbd8)

**Reply by jcard0na** [reply]
Nice PCB color!  Looking forward to see how the finished instrument looks!

**Reply by mikenturner** [reply]
Yes, I was going for a "bronze" saxophone colour, but it's not quite right.  But looks nice anyway

**Reply by jcard0na** [reply]
Seeing that this order was placed, I'm closing this.  Feel free to create another topic for another batch.  Thanks!
